site/zenodo_rdm/serializers/codemeta.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+"""Zenodo codemeta serializer."""
+
+from flask import current_app
+from flask_resources import BaseListSchema, MarshmallowSerializer
+from flask_resources.serializers import JSONSerializer
+from idutils import normalize_doi, to_url
+from invenio_rdm_records.contrib.codemeta.processors import CodemetaDumper
+from invenio_rdm_records.resources.serializers.codemeta.schema import CodemetaSchema
+from marshmallow import missing
+
+
+class ZenodoCodemetaSchema(CodemetaSchema):
+    """Zenodo Codemeta schema."""
+
+    id_ = missing
+
+    def get_id(self, obj):
```

Comment:
minor: switch to `get_identifiers` and keep `id` as it is in the original (pointing to DOI or `links.self`)